inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -120,7 +120,7 @@ def from_random_depths(
             random.uniform(-max_depth, 0),
         ]
 
-        _, _, depths = inductiva.generative.generate_grid_elevation(
+        _, _, depths = inductiva.generative.procedural.generate_random_terrain(
```

Comment:
see: here you are just storing the depths.
I really see no use case for the other two return values.
Please correct me if I am wrong.

Reply:
The others are necessary for other types of terrains. The bathymetry file just needs the depths, but other terrain types generate the mesh from the three variables.

Reply:
Why is that? Can you be more specific?
What do those variable hold that you don't already know before hand?

Reply:
In this case, they are generated inside the procedural.generate_random_terrain from x_range, y_range and x_num, y_num. They are the coordinates of all the points in the mesh and say respect to z-values we are generating.

Mesh files will be generated by the x_grid, y_grid and z_values.

Reply:
But you know those from the parameters you pass to the method, right?
Those are only related with scales. They don't have to be produced inside this method, or am i wrong?

Reply:
Indeed, we do. We require them to generate the Bathymetry and other terrain meshes. For the Bathymetry object they won't be need to thereafter run simulations (at least for now), but for the others they will.

Since, if we read a terrain from a file we already have them by default, this seems like the appropriate place to create them.